frontend/server/src/DAO/Courses.php

```diff
@@ -20,7 +20,7 @@ public static function findByName(string $name): array {
                 WHERE c.name
                 LIKE CONCAT('%', ?, '%') LIMIT 10";
 
-        /** @var list<array{acl_id: int, alias: string, course_id: int, description: string, finish_time: string, group_id: int, name: string, needs_basic_information: bool, public: bool, requests_user_information: string, school_id: int|null, show_scoreboard: bool, start_time: string}> */
+        /** @var list<array{acl_id: int, alias: string, course_id: int, description: string, finish_time: string|null, group_id: int, name: string, needs_basic_information: bool, public: bool, requests_user_information: string, school_id: int|null, show_scoreboard: bool, start_time: string}> */
```

Comment:
```suggestion
        /** @var list<array{acl_id: int, alias: string, course_id: int, description: string, finish_time: null|string, group_id: int, name: string, needs_basic_information: bool, public: bool, requests_user_information: string, school_id: int|null, show_scoreboard: bool, start_time: string}> */
```

desafortunadamente el checador de tipos de ese script necesita que la cadena concuerde exactamente :(